de/cheatsheet-deep-learning.md

```diff
@@ -1,162 +1,162 @@
 **1. Deep Learning cheatsheet**
 
-&#10230;
+&#10230; Deep Learning Spickzettel
 
 <br>
 
 **2. Neural Networks**
 
-&#10230;
+&#10230; Neuronale Netze
 
 <br>
 
 **3. Neural networks are a class of models that are built with layers. Commonly used types of neural networks include convolutional and recurrent neural networks.**
 
-&#10230;
+&#10230; Neuronale Netze sind eine Klasse von Modellen die in Schichten aufgebaut sind. Gängige Typen neuronaler Netze sind unter anderem faltende und rekurrente neuronale Netze.
 
 <br>
 
 **4. Architecture ― The vocabulary around neural networks architectures is described in the figure below:**
 
-&#10230;
+&#10230; Architektur - Das Vokabular rund um Architekturen neuronaler Netze ist in folgender Abbildung beschrieben: 
 
 <br>
 
 **5. [Input layer, hidden layer, output layer]**
 
-&#10230;
+&#10230; [Eingangsschicht, versteckte Schicht, Ausgangsschicht]
 
 <br>
 
 **6. By noting i the ith layer of the network and j the jth hidden unit of the layer, we have:**
 
-&#10230;
+&#10230; Sei i die i-te Schicht des Netzes und j die j-te versteckte Einheit der Schicht, so ist:
 
 <br>
 
 **7. where we note w, b, z the weight, bias and output respectively.**
 
-&#10230;
+&#10230; wobei w, b und z jeweils Gewichtung, Vorspannung und Ergebnis bezeichnen.
 
 <br>
 
 **8. Activation function ― Activation functions are used at the end of a hidden unit to introduce non-linear complexities to the model. Here are the most common ones:**
 
-&#10230;
+&#10230; Aktivierungsfunktion - Aktivierungsfunktionen werden am Ende einer versteckten Schicht benutzt um nicht-lineare Verflechtungen in das Model einzubringen. Hier sind die die gebräuchlisten:
```

Comment:
Aktivierungsfunktion - Aktivierungsfunktionen werden am Ende einer **verborgenen** Schicht benutzt**,** um nicht-lineare **Komplexität zu ermöglichen. Am häufigsten werden folgende Funktionen angewendet:**

"gebräuchlisten" is spelled incorrectly and rarely used.